install/updates/10-config.update

```diff
@@ -15,10 +15,8 @@ only: nsslapd-pluginPrecedence: 60
 
 # Set limits to suite better IPA deployment sizes, defaults are too
 # conservative
-dn: cn=config
-default: nsslapd-sizelimit:100000
-
-dn: cn=config,cn=ldbm database,cn=plugins,cn=config
```

Comment:
nsslapd-lookthroughlimit and nsslapd-idlistscanlimit are still documented as attributes of cn=config,cn=ldbm database,cn=plugins,cn=config, not of cn=default instance config,cn=chaining database,cn=plugins,cn=config.

Reply:
You're right, I should have left it in cn=config,cn=ldbm database,cn=plugins,cn=config. I got too cute in consolidating the changes